roles/common/templates/redis.conf.j2

```diff
@@ -0,0 +1,4 @@
+# {{ ansible_managed }}
+
+maxmemory 124mb
+maxmemory-policy volatile-lru
```

Comment:
for more details: https://redis.io/topics/lru-cache. LGTM :+1:

Reply:
I kept the same default max memory setting that we currently use for Memcached (124MB).

The **Volatile LRU** eviction strategy is there for the next step in this process, where we switch session storage from `ActiveRecord` to `Redis`. At that point this specific setting will be more important :+1:

Reply:
Basically it means if the cache store ever gets dangerously full, it will look for the most stale records first and evict them one by one as needed, so for example if you have a bunch of stale sessions that haven't been used in months, it'll drop the oldest ones first.